lib/data_update_scripts/20210316091354_move_authentication_settings.rb

```diff
@@ -0,0 +1,36 @@
+module DataUpdateScripts
+  class MoveAuthenticationSettings
+    AUTHENTICATION_SETTINGS = %w[
+      allow_email_password_login
+      allow_email_password_registration
+      allowed_registration_email_domains
+      apple_client_id
+      apple_key_id
+      apple_pem
+      apple_team_id
+      display_email_domain_allow_list_publicly
+      facebook_key
+      facebook_secret
+      github_key
+      github_secret
+      invite_only_mode
+      providers
```

Comment:
IIRC this is called `authentication_providers` in `SiteConfig`, thus it's not copying the correct value over

https://github.com/forem/forem/blob/4848a8b2e17b3c09e921acb01f0e0c5b3bcf115d/app/models/site_config.rb#L40